RapidReact/src/main/java/frc/robot/config/Config.java

```diff
@@ -161,7 +161,7 @@ public Config() {
     climber.climberLeader.positionPIDF = new PIDF(/*P*/0.1, /*I*/0, /*D*/0, /*F*/0.00018);
     // Configure output and sensor direction 
     climber.climberLeader.inverted = false;
-    climber.climberLeader.sensorPhase = true;
+    climber.climberLeader.sensorPhase = false;
```

Comment:
Rename these to climberLeft and climberRight or something. It isn't fair to call one the leader and the other the follower, since they both follow their setpoints independently, in addition to an auxilliary control loop that moves both of them towards each other if they aren't in the same place.

Reply:
I think in the current commit the follower does indeed follow the leader entirely -- but it also wasn't moving at all when we tested it sooo we might have to add its own (aux) pidloop back.